src/Index.php

```diff
@@ -136,4 +162,72 @@ private function loadConfig(array $indexConfig)
         }
     }
 
+    /**
+     * Returns the alias of the current index
+     *
+     * @return string
+     */
+    public function getSearchIndexName(): string
+    {
+        return $this->name . '_alias';
+    }
+
+    public function getMainIndexName(): string
+    {
+        return $this->name;
+    }
+
+    private function getTmpIndexName(): string
+    {
+        return $this->name . '_tmp';
+    }
+
+    private function setMainAsAlias(): array
+    {
+        $this->removeParamsAsAlias($this->getTmpAliasParams());
+        return $this->setParamsAsAlias($this->getMainAliasParams());
+    }
+
+    private function setTmpAsAlias(): array
+    {
+        $this->removeParamsAsAlias($this->getMainAliasParams());
+        return $this->setParamsAsAlias($this->getTmpAliasParams());
+    }
+
+    private function getMainAliasParams(): array
+    {
+        return [
+            'index' => $this->getMainIndexName(),
+            'name' => $this->getSearchIndexName()
+        ];
+    }
+
+    private function getTmpAliasParams(): array
+    {
+        return [
+            'index' => $this->getTmpIndexName(),
+            'name' => $this->getSearchIndexName()
+        ];
+    }
+
+    private function removeParamsAsAlias(array $params): void
```

Comment:
The `void`  return type is not supported by PHP 7.0, which is supported by the client.

I'm in favor of removing the `void` return type to keep the 7.0 compatibility. 

```suggest
private function removeParamsAsAlias(array $params)
```